dev/test/bundle.ts

```diff
@@ -128,11 +131,14 @@ describe('Bundle Buidler', () => {
     expect(elements.length).to.equal(4);
 
     const meta = (elements[0] as IBundleElement).metadata;
```

Comment:
Do you mind adding a helper for the metadata comparison?

Reply:
Done.